Source/Core/Common/Crypto/SHA1.h

```diff
@@ -24,6 +24,12 @@ class Context
 public:
   virtual ~Context() = default;
   virtual void Update(const u8* msg, size_t len) = 0;
+  void Update(const std::vector<u8>& msg) { return Update(msg.data(), msg.size()); }
+  template <size_t length>
+  void Update(const std::array<u8, length>& data)
+  {
+    return Update(data.data(), data.size());
+  }
```

Comment:
You can remove these, the `std::span` overload just handles them.

Reply:
I looked it up and wasn't convinced, guess I should have just tried it.